LocalPackages/SubscriptionUI/Sources/SubscriptionUI/DebugMenu/SubscriptionDebugMenu.swift

```diff
@@ -92,6 +93,12 @@ public final class SubscriptionDebugMenu: NSMenuItem {
         environmentItem.submenu = makeEnvironmentSubmenu()
         menu.addItem(environmentItem)
 
+        menu.addItem(.separator())
+        let storefrontID = SKPaymentQueue.default().storefront?.identifier ?? "nil"
+        menu.addItem(NSMenuItem(title: "Storefront ID: \(storefrontID)", action: nil, target: nil))
+        let storefrontCountryCode = SKPaymentQueue.default().storefront?.countryCode ?? "nil"
```

Comment:
This API was deprecated in macOS 15, but it'll be a long time until we're dropping support for it so we're safe to use it here. This API is easier for this use that then new `async` option, but please let me know if you see an issue with using it.

Reply:
Makes sense 👍